packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -206,7 +203,47 @@ describe('Line chart rendering', () => {
       expect(container).toMatchSnapshot();
     },
     undefined,
-    beforeAll,
+    undefined,
+    !(isTimezoneSet(Timezone.UTC) && env === 'TEST'),
```

Comment:
Extract the TEST env check as a test utility function so that the env variable doesn't need to be imported in every test file, making it easier to modify the check in the future if required.